tests/Jellyfin.Naming.Tests/Video/CleanStringTests.cs

```diff
@@ -1,133 +1,45 @@
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestBDrip()
-        {
-            Test("Crouching.Tiger.Hidden.Dragon.BDrip.mkv", "Crouching.Tiger.Hidden.Dragon");
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestBDripHDC()
-        {
-            Test("Crouching.Tiger.Hidden.Dragon.BDrip-HDC.mkv", "Crouching.Tiger.Hidden.Dragon");
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestMulti()
-        {
-            Test("Crouching.Tiger.Hidden.Dragon.4K.UltraHD.HDR.BDrip-HDC.mkv", "Crouching.Tiger.Hidden.Dragon");
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestLeadingBraces()
-        {
-            // Not actually supported, just reported by a user
-            Test("[0004] - After The Sunset.el.mkv", "After The Sunset");
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestTrailingBraces()
-        {
-            Test("After The Sunset - [0004].mkv", "After The Sunset");
-        }
-
-        private void Test(string input, string expectedName)
-        {
-            var result = GetParser().CleanString(input).ToString();
-
-            Assert.Equal(expectedName, result, true);
+        private readonly NamingOptions _namingOptions = new NamingOptions();
+
+        [Theory]
+        [InlineData("Super movie 480p.mp4", "Super movie")]
+        [InlineData("Super movie 480p 2001.mp4", "Super movie")]
+        [InlineData("Super movie [480p].mp4", "Super movie")]
+        [InlineData("480 Super movie [tmdbid=12345].mp4", "480 Super movie")]
+        [InlineData("Super movie(2009).mp4", "Super movie(2009).mp4")]
+        [InlineData("Run lola run (lola rennt) (2009).mp4", "Run lola run (lola rennt) (2009).mp4")]
+        [InlineData(@"American.Psycho.mkv", "American.Psycho.mkv")]
+        [InlineData(@"American Psycho.mkv", "American Psycho.mkv")]
+        [InlineData(@"[rec].mkv", "[rec].mkv")]
+        [InlineData("Crouching.Tiger.Hidden.Dragon.4k.mkv", "Crouching.Tiger.Hidden.Dragon")]
+        [InlineData("Crouching.Tiger.Hidden.Dragon.UltraHD.mkv", "Crouching.Tiger.Hidden.Dragon")]
+        [InlineData("Crouching.Tiger.Hidden.Dragon.UHD.mkv", "Crouching.Tiger.Hidden.Dragon")]
+        [InlineData("Crouching.Tiger.Hidden.Dragon.HDR.mkv", "Crouching.Tiger.Hidden.Dragon")]
+        [InlineData("Crouching.Tiger.Hidden.Dragon.HDC.mkv", "Crouching.Tiger.Hidden.Dragon")]
+        [InlineData("Crouching.Tiger.Hidden.Dragon-HDC.mkv", "Crouching.Tiger.Hidden.Dragon")]
+        [InlineData("Crouching.Tiger.Hidden.Dragon.BDrip.mkv", "Crouching.Tiger.Hidden.Dragon")]
+        [InlineData("Crouching.Tiger.Hidden.Dragon.BDrip-HDC.mkv", "Crouching.Tiger.Hidden.Dragon")]
+        [InlineData("Crouching.Tiger.Hidden.Dragon.4K.UltraHD.HDR.BDrip-HDC.mkv", "Crouching.Tiger.Hidden.Dragon")]
+        // FIXME: [InlineData("After The Sunset - [0004].mkv", "After The Sunset")]
```

Comment:
what's broken with this one?

Reply:
It doesn’t strip “ -“

Reply:
We should probably make an issue about that test case then.

Reply:
Maybe...